bin/update-all-the-pipelines

```diff
@@ -5,6 +5,22 @@ require 'yaml'
 
 raise '`brew install lastpass-cli` is required' if `which lpass` == ''
 
+LPASS_CONCOURSE_PRIVATE = ENV['LPASS_CONCOURSE_PRIVATE_FILE'] || 'Shared-Buildpacks/concourse-private.yml'
+LPASS_DEPLOYMENTS_BUILDPACKS = ENV['LPASS_DEPLOYMENTS_BUILDPACKS_FILE'] || 'Shared-Buildpacks/deployments-buildpacks.yml'
+LPASS_REPOS_PRIVATE_KEYS = ENV['LPASS_REPOS_PRIVATE_KEYS_FILE'] || 'Shared-Buildpacks/buildpack-repos-private-keys.yml'
+
+public_config = YAML.load_file("public-config.yml")
+
+lpass_config= {}
+lpass_yaml_data=%x{lpass show #{LPASS_CONCOURSE_PRIVATE} --notes && lpass show #{LPASS_DEPLOYMENTS_BUILDPACKS} --notes}
+if $?.exitstatus != 0
+  puts "WARNING: ignoring lastpass config file. An error occured while processing #{LPASS_CONCOURSE_PRIVATE} and #{LPASS_DEPLOYMENTS_BUILDPACKS}"
+else
+  lpass_config = YAML.load(lpass_yaml_data)
+end
+
+full_config = public_config.merge(lpass_config)
```

Comment:
What is the purpose of this `full_config` variable and the merge? It looks like only `domain-name` is only fetched from this `full_config`, and `domain-name` is specified in the public config, so I don't see why the `public_config` needs to be merged with the `lpass_config`?